tool/tctl/common/top_command.go

```diff
@@ -879,6 +883,24 @@ func getRemoteClusters(metric *dto.MetricFamily) []RemoteCluster {
 	return out
 }
 
+func getActiveMigrations(metric *dto.MetricFamily) []string {
+	if metric == nil || metric.GetType() != dto.MetricType_GAUGE || len(metric.Metric) == 0 {
+		return nil
+	}
+	var out []string
+	for _, counter := range metric.Metric {
+		if counter.Gauge.GetValue() == 0 {
+			continue
+		}
+		for _, label := range counter.Label {
+			if label.GetName() == "name" {
+				out = append(out, label.GetValue())
```

Comment:
can there be more than one `name` label? If not we can break here

Reply:
Thanks! Done in f26f8731a0edc2b1ad7eb0e72d651c82a3be57bc.